docs/src/main/sphinx/admin/properties-general.rst

```diff
@@ -54,3 +54,22 @@ remove usage of this property.
 Ensure to use this only as a temporary measure to assist in your migration
 efforts.
 
+``protocol.v1.prepared-statement-compression.length-threshold``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-integer`
+* **Default value:** ``2048``
+
+Prepared statements that are submitted to Trino for processing, and are longer
+than the value of this property, are compressed for transport via the HTTP
+header to improve handling and to avoid failures due to hitting HTTP header size
+limits.
+
+``protocol.v1.prepared-statement-compression.min-gain``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-integer`
+* **Default value:** ``512``
+
+Prepared statement compression is not applied if the size gain is less than the
+configured value. Smaller statements do not benefit occur from compression.
```

Comment:
> do not benefit occur

something is off

Reply:
Something went wrong during my rewording .. updated now..